site/_docs/file-tail.md

```diff
@@ -249,12 +249,25 @@ For booleans, the statistics include the count of false and true values.
 }
 ```
 
-For decimals, the minimum, maximum, and sum are stored.
+For decimals, the minimum, maximum, and sum are stored. In ORC 2.0,
+string representation is deprecated and DecimalStatistics uses integers
+which have better performance.
 
 ```message DecimalStatistics {
  optional string minimum = 1;
  optional string maximum = 2;
  optional string sum = 3;
+  message Int128 {
+   repeated sint64 highBits = 1;
+   repeated uint64 lowBits = 2;
```

Comment:
shouldn't this be sint64 as well since we are using uint64 for the SECONDARY stream?

Reply:
Here I was aligning with C++ orc::Int128's implementation to avoid many casts.